torch/testing/_internal/opinfo/definitions/fft.py

```diff
@@ -217,7 +221,18 @@ def mt(shape, **kwargs):
                 precisionOverride({torch.float: 2e-4, torch.cfloat: 2e-4}),
                 "TestFFT",
                 "test_reference_nd",
-            )
+            ),
+            DecorateInfo(
+                toleranceOverride(
+                    {
+                        torch.float16: tol(atol=1e-4, rtol=1e-4),
+                        torch.complex64: tol(atol=1e-4, rtol=1e-4),
```

Comment:
With the new tolerances, all these can be removed.

Reply:
done